app/indices/conversation_index.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Elasticsearch
+class ConversationIndex
```

Comment:
I don't see how indices getting created in db.
It should be created when record is created/updated/deleted